assembl/views/api2/votes.py

```diff
@@ -196,3 +200,56 @@ def vote_results_csv(request):
     ctx._instance.csv_results(output, histogram)
     output.seek(0)
     return Response(body_file=output, content_type='text/csv')
+
+
+@view_config(context=InstanceContext, request_method='GET',
+             ctx_instance_class=VotingWidget,
+             name="vote_results_csv", permission=P_DISC_STATS)
+def global_vote_results_csv(request):
+    ctx = request.context
+    user_id = request.authenticated_userid
+    if not user_id:
+        raise HTTPUnauthorized
+    widget = ctx._instance
+    if widget.activity_state != "ended":
```

Comment:
The activity state isn't an `enum` already?

Reply:
nope. good idea, but in scope?